AppKit/CPView.j

```diff
@@ -239,6 +240,8 @@ var CPViewHighDPIDrawingEnabled = YES;
     BOOL                _allowsVibrancy         @accessors(property=allowsVibrancy);
     CPAppearance        _appearance             @accessors(getter=appearance);
     CPAppearance        _effectiveAppearance;
+
+    CPMutableArray      _trackingAreas          @accessors(getter=trackingAreas);
```

Comment:
This is supposed to return a copy, otherwise it isn't actually read-only. So it's `@accessors(getter=trackingAreas, copy)`.